modules/@apostrophecms/schema/ui/apos/components/AposInputString.vue

```diff
@@ -96,6 +96,12 @@ export default {
           return 'max';
         }
       }
+      if (this.field.type === 'email') {
+        const matches = value.match(/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/);
```

Comment:
Let's note where the regex came from here as well.

Reply:
Done

Reply:
This isn't checked by eslint for some reason, but could be updated to match the passing pattern from the other instance. 

`/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/`